types/tizen-common-web/test/humanactivity-test.ts

```diff
@@ -0,0 +1,46 @@
+import { HumanActivityAccumulativePedometerData, HumanActivityPedometerData, HumanActivitySleepMonitorData, HumanActivityType } from "tizen-common-web/humanactivity";
+import { WebAPIError } from "tizen-common-web/tizen";
+
+{
+    function onchangedCB(sleepInfo?: HumanActivitySleepMonitorData): void {
```

Comment:
Note that the types above don't constrain the parameter type (except that it must be an object).  So, it could be `sleepInfo?: { asExpected: false }`, for example.